src/v2i-hub/MUSTSensorDriverPlugin/scripts/MockMUSTSensor.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python3
+
+import socket
+import sys
+import time
+import argparse
+from dataclasses import dataclass
+from enum import Enum
+
+
+class DetectionClassification(Enum):
+    """Enumeration used for indentifying type of detection
+    """
+    SEDAN='sedan'
+    VAN='van'
+    TRUCK='truck'
+class DetectionSize(Enum):
+    """Enumeration used for indentifying the type of KafkaLogMessage
+    """
+    SMALL='small'
+    MEDIUM='medium'
+    LARGE='large'
+
+@dataclass
+class MUSTDetection:
+    """Class used to store data for each Kafka Log Message
+    """
+    classification: DetectionClassification
+    x: float
+    y: float
+    heading: float
+    speed: float
+    size: DetectionSize
+    confidence: float
+    track_id: int
+    timestamp: int
+
+    def to_csv(self):
+        return f'{self.classification.value},{self.x},{self.y},{self.heading},{self.speed},{self.size.value},{self.confidence},{self.track_id},{self.timestamp}'
+
+def move_detection():
+    return 
+
+def create_socket():       
+    try:
+        return socket.socket(family=socket.AF_INET, type=socket.SOCK_DGRAM)
+    except socket.error as err:
+        print('Socket error because of %s' %(err))
+def send_detection(sock, detection, host):
+    try:
+        msg = detection.to_csv()
+        encoded_msg = str.encode(msg)
+        sock.sendto(encoded_msg,host)
+        print( encoded_msg.decode(encoding= 'UTF-8'), 'was sent to ', host)
+    except socket.gaierror:
+        print ('There an error resolving the host')
+
+detections = []
+detections.append(MUSTDetection(DetectionClassification.SEDAN, 0, 0, 330, 1, DetectionSize.MEDIUM, 95, 2,round(time.time())))
+
+def main():
+    parser = argparse.ArgumentParser(description='Script to mock detection data coming from MUST Sensor')
+    parser.add_argument('--ip', help='IP address to send detection data to.', type=str, default="127.0.0.1") 
+    parser.add_argument('--port', help='Port to send detection data to.', type=str, default=4545)  
+    args = parser.parse_args()
+    sock = create_socket()
```

Comment:
Maybe exit or retry if create_socket failed?

Reply:
This is not a connection, call does not have a documented failure case. It is initializing the socket (https://docs.python.org/3/library/socket.html#functions)